pybossa/view/account.py

```diff
@@ -499,6 +502,15 @@ def create_account(user_data, project_slugs=None, ldap_disabled=True, auto_creat
                      password=user_data['password'])
     msg = generate_invitation_email_for_new_user(user=user_info, project_slugs=project_slugs)
     mail_queue.enqueue(send_mail, msg)
+    data_access = user_data.get('data_access', [])
+    # if the account is created automatically and has L4 data access, send warning
+    if auto_create and 'L4' in data_access:
+        admin_msg = generate_bsso_account_notification(user=user_data, admins_emails=current_app.config.get('ADMINS',[]), access_type="BSSO", warning=True)
+        mail_queue.enqueue(send_mail, admin_msg)
+    # if the account is just automatically created, just send notification
+    elif auto_create:
+        alert_msg = generate_bsso_account_notification(user=user_info, admins_emails=current_app.config['ALERT_LIST'], access_type="BSSO")
```

Comment:
very minor, consider 
```
alert_msg = generate_bsso_account_notification(user=user_info, admins_emails=current_app.config('ALERT_LIST', []), access_type="BSSO")
```
in case the ALERT_LIST doesn't exist

Reply:
curious, will the empty recipient list cause any problem in mail queue?

Reply:
It doesn't seem to. I added a bogus value for the admins_email key and then ran all the web tests that test the email queue and everything passed.